AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -118,22 +118,6 @@ - (CGSize)size
   return _calculatedSize;
 }
 
-- (void)setConstrainedSize:(CGSize)constrainedSize
```

Comment:
Currently just for robustness we remove setting the constrain size explicitly as the lookup for a renderer in the cache should be pretty fast. We have to look at the performance though and optimize afterwards if necessary though.